libnm-core/nm-core-internal.h

```diff
@@ -101,6 +101,11 @@
 #define NM_BR_MIN_AGEING_TIME   0
 #define NM_BR_MAX_AGEING_TIME   1000000
 
+#define NM_BR_PORT_MAX_PRIORITY 63
```

Comment:
typo in commit message (a spurious space at `move bridge port min/max/default values  to core-internal`).

Reply:
Fixed, thanks

Reply:
fixed, thanks